theano/gof/op.py

```diff
@@ -1168,11 +1168,19 @@ def test_gxx_support():
         }
 }
         """
-        default_openmp = GCC_compiler.try_compile_tmp(
-            src_code=code,
-            tmp_prefix='test_omp_',
-            flags=['-fopenmp'],
-            try_run=False)
+        # TODO: need to resolve different openmp flags
+        if config.cxx == "icpc":
+            default_openmp = CXX_compiler.try_compile_tmp(
+                src_code=code,
+                tmp_prefix='test_omp_',
+                flags=['-qopenmp'],
+                try_run=False)
+        else:
+            default_openmp = CXX_compiler.try_compile_tmp(
+                src_code=code,
+                tmp_prefix='test_omp_',
+                flags=['-fopenmp'],
+                try_run=False)
```

Comment:
The only difference is the flags parameter. Can you just set a temp variable flags=[...] in both branch of the code and call try_compile_tmp() once after the if. This make the code more readable.